src/multibody/fcl.hpp

```diff
@@ -92,7 +94,7 @@ struct GeometryObject
   /// \brief Index of the parent joint
   JointIndex parentJoint;
 
-  /// \brief The actual cloud of points representing the collision mesh of the object after scaling.
+  /// \brief The FCL CollisionGeometry
```

Comment:
It might be worth saying that *meshFile + meshScale -> fcl* as a note.

Reply:
What do you mean here?

Reply:
I see two interpretation of the 3 attributes meshFile, meshScale and fcl:
1. fcl == meshFile (i.e. the scale wasn't applied when loading),
2. fcl == meshFile + meshScale (i.e. the scale was applied when loading).

For example, if meshFile contains a line of 1 meter, and meshScale is 0.1, then:
1. fcl contains a line of 1 meter,
2. fcl contains a line of 0.1 meter,